src/subdomains/generic/kyc/entities/name-check-log.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { UpdateResult } from 'src/shared/models/entity';
+import { ChildEntity, Column, ManyToOne } from 'typeorm';
+import { BankData } from '../../user/models/bank-data/bank-data.entity';
+import { KycLog } from './kyc-log.entity';
+
+export enum RiskStatus {
+  SANCTIONED = 'Sanctioned',
+  NOT_SANCTIONED = 'NotSanctioned',
+}
+
+export enum ManualRiskRate {
+  CONFIRMED = 'Confirmed',
+  IGNORED = 'Ignored',
+  NOT_MATCHING = 'NotMatching',
+}
```

Comment:
Maybe we should call these enums `RiskStatus` and `RiskAssessment` (or `RiskEvaluation`)?